client/packages/common/src/ui/layout/tables/components/Cells/CurrencyCell.tsx

```diff
@@ -28,12 +28,12 @@ export const CurrencyCell = <T extends RecordWithId>({
 > => {
   const { c } = useCurrency(currencyCode);
   const price = Number(column.accessor({ rowData }) ?? 0);
-  // format prices > 1 with default precision
-  const precision = price < 1 ? 10 : undefined;
-  const fullText = c(price, precision).format();
+  const fullText = c(price, 10).format();
   let text = fullText;
-  if (price !== 0 && price < 0.001) {
-    text = `< ${c(0.001, 3).format()}`;
+  if ((price * 100) % 1 !== 0) {
```

Comment:
A comment here might be nice?

Reply:
```suggestion
// if the price has more than 2 decimal places, round to 2 DP and add ellipsis, if less than 0.01 just show "<0.01"
  if ((price * 100) % 1 !== 0) {
```

Reply:
Comment in connecting PR (:

Reply:
https://github.com/msupply-foundation/open-msupply/pull/4939